nginx/cernopendata.conf

```diff
@@ -22,16 +31,38 @@ server {
         root /usr/local/var/cernopendata/var/cernopendata-instance;
     }
 
-    location / {
-        proxy_pass http://web:5000;
+    location ~* /eos/opendata/.*\.(root|tgz|zip)$ {
+        proxy_cache off;
+        proxy_set_header X-Real-IP $remote_addr;
         proxy_set_header Host $host;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_pass https://eospublichttp.cern.ch;
+    }
+
+    # Cache any URL that has '/files/' pattern in it.
+    # (e.g. '/records/123/files/abc', '/dataset/test/files/storage/higgs.txt')
+    location ~ ^/.+/files/.+ {
+        proxy_cache cache_cod_files;
+        proxy_cache_key $host$uri$is_args$args;
+        proxy_cache_valid 200 301 302 30m;
+        expires 1h;
         proxy_set_header X-Real-IP $remote_addr;
+        proxy_set_header Host $host;
         proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_ignore_headers Set-Cookie;
+        proxy_pass http://web:5000;
     }
 
-    error_page   500 502 503 504  /50x.html;
-    location = /50x.html {
-        root   /usr/share/nginx/html;
+    location / {
+        proxy_cache cache_cod;
+        proxy_cache_key $host$uri$is_args$args;
+        proxy_cache_valid 200 301 302 30m;
+        expires 1h;
+        proxy_set_header X-Real-IP $remote_addr;
+        proxy_set_header Host $host;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_ignore_headers Set-Cookie;
```

Comment:
OK to remove cookies related to searching vs faceting?

Reply:
Yes